pandas/tests/extension/test_categorical.py

```diff
@@ -207,6 +207,21 @@ def test_cast_nan_to_int(self, cls, values):
         with pytest.raises((ValueError, TypeError), match=msg):
             s.astype(int)
 
+    @pytest.mark.parametrize(
+        "values",
```

Comment:
small nitpick but a little more direct to just call this `"expected"` as you immediately do that renaming in the test